modules/calcite/src/main/java/org/apache/ignite/internal/processors/query/calcite/exec/exp/agg/Accumulators.java

```diff
@@ -973,7 +973,7 @@ private DecimalMinMax(AggregateCall aggCall, RowHandler<Row> hnd, boolean min) {
         private final boolean min;
 
         /** */
-        private final Function<IgniteTypeFactory, RelDataType> typeSupplier;
+        private final transient Function<IgniteTypeFactory, RelDataType> typeSupplier;
```

Comment:
Is it really required? Looks like accumulators is not serialized

Reply:
Fixed